docs/source/yelpsoa_configs.rst

```diff
@@ -112,6 +112,10 @@ instance MAY have:
     * ``PAASTA_INSTANCE``: The instance name
     * ``PAASTA_CLUSTER``: The cluster name
 
+    Marathon adds a ``MARATHON_`` prefix to environment variables it sets up to
```

Comment:
This reads a bit weird. How about:

Additionally there are MARTHON_ prefixed variables available. See (doc) for more information about these variables.